org.scintilla.SciTE.yaml

```
@@ -0,0 +1,46 @@
+app-id: org.scintilla.SciTE
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: SciTE
+rename-desktop-file: SciTE.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=host
+  
+modules:
+
+- name: SciTE
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://www.scintilla.org/scite446.tgz
+    strip-components: 0
+    sha256: 1c2e6eee6c8aa2e52983d9713a4b1c97d06c376324a3ec8932adfcb9e55d16d1
+  - type: file
+    path: org.scintilla.SciTE.metainfo.xml
+  - type: file
+    path: org.scintilla.SciTE.png
+    
+  build-commands:
+  - cd scintilla/gtk && 
+       make GTK3=1 -j $FLATPAK_BUILDER_N_JOBS prefix="${FLATPAK_DEST}"
+  - cd scite/gtk && 
+       make GTK3=1 -j $FLATPAK_BUILDER_N_JOBS prefix="${FLATPAK_DEST}" && 
+       make GTK3=1 install prefix="${FLATPAK_DEST}"
+  
+  - mkdir -p /app/share/metainfo
+  - install -Dm644 org.scintilla.SciTE.metainfo.xml /app/share/metainfo/
+  
+  - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/
+  - install -Dm644 org.scintilla.SciTE.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/
+
+  post-install:
+  - ln -sf ${FLATPAK_DEST}/lib/scite/*.so 
+           ${FLATPAK_DEST}/lib
+  - sed 's/Icon=Sci48M/Icon=org.scintilla.SciTE/g' -i 
+    ${FLATPAK_DEST}/share/applications/SciTE.desktop
+
```

```suggestion
  - ln -sf ${FLATPAK_DEST}/lib/scite/*.so 
           ${FLATPAK_DEST}/lib
  - sed 's/Icon=Sci48M/Icon=org.scintilla.SciTE/g' -i 
    ${FLATPAK_DEST}/share/applications/SciTE.desktop
```

No need to use `post-install` with the simple buildsystem.